app/editor/src/features/content/tool-bar/sections/filter/CreateNewSection.tsx

```diff
@@ -45,7 +45,7 @@ export const CreateNewSection: React.FC = () => {
           </Row>
         </Col>
       }
-      label="CREATE NEW"
+      label="CREATE SNIPPET"
```

Comment:
A snippet is used to express a Radio/TV content type.  I'm not certain it makes sense to call all new content a snippet.  Was this Bobbi's suggestion?

Reply:
Yeah this was after talking to her, but I can remove for now and maybe get some clarification from Bobbi and Carolynn?

Reply:
Currently chatting with Bobbi - will update you when I hear back

Reply:
Response: "They don't use snippet just for radio and tv, it's any content"